curriculum/challenges/english/01-responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast.md

```diff
@@ -54,7 +54,7 @@ assert(code.match(/background-color:\s*?hsl\(120,\s*?25%,\s*?55%\)/gi));
   </header>
   <article>
     <h2>A Word on the Recent Catnip Doping Scandal</h2>
-    <p>The influence that catnip has on feline behavior is well-documented, and its use as an herbal supplement in competitive ninja circles remains controversial. Once again, the debate to ban the substance is brought to the public's attention after the high-profile win of Kittytron, a long-time proponent and user of the green stuff, at the Claw of Fury tournament.</p>
+    <p>The influence that catnip has on feline behavior is well-documented, and its use as a herbal supplement in competitive ninja circles remains controversial. Once again, the debate to ban the substance is brought to the public's attention after the high-profile win of Kittytron, a long-time proponent and user of the green stuff, at the Claw of Fury tournament.</p>
```

Comment:
Whether to use "an" or "a" depends on the sound the following word begins with. I think "herb" is sometimes pronounced with an "h" sound at the beginning (I believe this is the way they often pronounce it in the UK) which would mean that it should be preceded by "a". But in the US I believe that most people pronounce it "erb" (without the "h" sound) and thus it would be preceded by "an". So perhaps there is no one correct answer and this can go either way. But since Quincy lives in Texas then I propose we assume the American pronunciation and keep it "an" :-)

Reply:
Ah, good point! I'm from the UK so to me it sounded weird for the reasons you explained. I'll revert that change :-)

Reply:
I wasn't being 100% completely serious here. I'm not necessarily opposed to changing it. I guess I just felt like it was perhaps an unnecessary change and thus was having some fun with it.